style.css

```diff
@@ -400,3 +400,214 @@ form input,
   background-color: rgba(255, 255, 255, 0.3);
   border: rgba(255, 255, 255, 1);
 }
+
+@media screen and (min-width: 992px) {
+  #navbar {
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding: 20px 0;
+    background-color: #fff;
+  }
+
+  .links {
+    display: block;
+    background: none;
+    backdrop-filter: blur(0);
+    padding: 0;
+    mix-blend-mode: normal;
+    overflow: visible;
+    width: 100%;
+    height: 0;
+  }
+
+  #hamBtn {
+    display: none;
+  }
+
+  #hamBtnClose {
+    display: none;
+  }
+
+  .links li {
+    margin: 0 30px;
+  }
+
+  #toolbar {
+    margin-left: 155px;
+  }
+
+  .nav__links {
+    margin-top: 20px;
+    padding-right: 155px;
+    display: flex;
+    justify-content: flex-end;
+  }
+
+  .nav__links a {
+    color: #344563;
+    font-size: 15px;
+    font-weight: 500;
+  }
+
+  .background {
+    background-image: url(imgs/mobile-background.svg);
+    background-color: #fff;
+    background-repeat: no-repeat;
+    background-size: 90%;
+    height: 100%;
+    border-radius: 0 0 0 110px;
+  }
+
+  .headline {
+    display: flex;
+    flex-direction: column;
+    padding: 150px 260px 0 260px;
+  }
+
+  .headline h1 {
+    font-size: 48px;
+    line-height: 60px;
+  }
+
+  .headline p {
+    padding-right: 350px;
+  }
+
+  .hl-paragraph {
+    padding-right: 110px;
+    font-size: 20px;
+    font-weight: 300;
+  }
+
+  .hl-button {
+    width: fit-content;
+  }
+
+  .work-1 {
+    margin-top: 100px;
+  }
+
+  #portfoliosection {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .card-work {
+    display: flex;
+    width: 1170px;
+    height: 445px;
+  }
+
+  .work-1,
+  .work-2,
+  .work-3,
+  .work-4 {
+    width: auto;
+  }
+
+  .card-work:hover {
+    cursor: pointer;
+  }
+
+  .img-card {
+    width: 60%;
+    display: flex;
+  }
+
+  .img-card img {
+    width: 100%;
+    height: 100%;
+    object-fit: contain;
+  }
+
+  .about-project {
+    margin: 0 0 0 46px;
+    flex-direction: column;
+  }
+
+  .about-project p {
+    margin-bottom: 24px;
+  }
+
+  .tag {
+    margin-bottom: 0;
+  }
+
+  .about {
+    display: flex;
+    flex-direction: row;
+    padding: 142px 142px;
+  }
+
+  .about-content {
+    width: 40%;
+    margin-left: 160px;
+  }
+
+  .about h2 {
+    margin-top: 0;
+    padding-top: 0;
+  }
```

Comment:
- [ ] Please also as required in the project requirements, you must stick to the design as much as possible (margins top, bottom, right, and left, font and font sizes) using the Figma Template provided. Please try to align the title and the description in the about section to the left. 
- [ ] Kindly add the images of languages as provided in the Figma design.
- Your design:

<img width="948" alt="image" src="https://user-images.githubusercontent.com/77190203/155863441-34cded3d-8fdc-46fe-962b-4a9aec1462ba.png">

- Figma design